### PR TITLE
Update go.mod jsonpatch v3

### PR DESCRIPTION
**Pull request by port22**
Update go.mod jsonpatch v3

related to: https://github.com/jpillora/webproc/issues/18

### CODE REVIEW

File: go/state.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 	"sync/atomic"
 	"time"
 
-	"gomodules.xyz/jsonpatch/v2"
+	"gomodules.xyz/jsonpatch/vv3"
```
Comment (jpillora) [inline go/state.go]: vv3?

Comment (port22) [reply go/state.go]: sry typo, fixed

**Comment by jpillora** [comment]
Thanks!

**Comment by jpillora** [comment]
released as `v0.4.0` 👍

**Pull request merged**